tutordiscovery/templates/discovery/build/discovery/Dockerfile

```diff
@@ -62,7 +67,10 @@ RUN {% if is_buildkit_enabled() %}--mount=type=cache,target=/openedx/.cache/pip,
     # uwsgi server https://pypi.org/project/uWSGI/
     uwsgi==2.0.21
 
-# Collect static assets
+# Update i18n and collect static assets
+{% if DISCOVERY_ATLAS_PULL %}
+RUN make OPENEDX_ATLAS_PULL=yes ATLAS_PULL_LANGS="{{ DISCOVERY_ATLAS_PULL_LANGS }}" pull_translations
```

Comment:
- The `DISCOVERY_ATLAS_PULL_LANGS` setting is undefined. How can we make Atlas automatically (or at least: by default) download strings from all "sufficiently translated" languages? As a rule of thumb, we like to include in Tutor all languages with >50% reviewed strings.
- In Palm, the `make pull_translations` command is going to run `tx`, and not `atlas`. Actually, neither does the master branch run Atlas. 
- Also, how are we going to pull translations from Transifex (I assume?) if there is no transifexrc file in the Docker image?
- Actually, do we really have to run a `make` command? Can we just run `atlas` directly to remove one level of indirection?
- Once we have a basic version working, we should try to move the atlas step to a separate Docker layer.